editor/editor_about.cpp

```diff
@@ -37,13 +37,34 @@
 #include "editor/gui/editor_version_button.h"
 #include "editor/themes/editor_scale.h"
 #include "scene/gui/item_list.h"
+#include "scene/gui/rich_text_label.h"
+#include "scene/gui/scroll_container.h"
 #include "scene/gui/separator.h"
 #include "scene/gui/split_container.h"
 #include "scene/gui/tab_container.h"
+#include "scene/gui/texture_rect.h"
+#include "scene/gui/tree.h"
 #include "scene/resources/style_box.h"
 
 void EditorAbout::_notification(int p_what) {
 	switch (p_what) {
+		case NOTIFICATION_TRANSLATION_CHANGED: {
+			_about_text_label->set_text(
+					String::utf8("\xc2\xa9 2014-present ") + TTR("Godot Engine contributors") + "." +
+					String::utf8("\n\xc2\xa9 2007-2014 Juan Linietsky, Ariel Manzur.\n"));
```

Comment:
You could improve this if it's moved anyway:
```suggestion
					String(U"© 2014-present ") + TTR("Godot Engine contributors") + ".\n" +
					String(U"© 2007-2014 Juan Linietsky, Ariel Manzur.\n"));
```